src/java/org/apache/cassandra/index/sai/disk/v2/sortedterms/SortedTermsReader.java

```diff
@@ -126,6 +126,25 @@ public long getPointId(@Nonnull ByteComparable term)
         }
     }
 
+    public long getLastPointId(@Nonnull ByteComparable term)
+    {
+        Preconditions.checkNotNull(term, "term null");
+
+        try (TrieRangeIterator reader = new TrieRangeIterator(termsTrie.instantiateRebufferer(),
+                                                              meta.trieFP,
+                                                              term,
+                                                              null,
+                                                              true,
+                                                              true))
+        {
+            final Iterator<Pair<ByteSource, Long>> iterator = reader.iterator();
+            long pointId = Long.MAX_VALUE;
+            while (iterator.hasNext())
+                pointId = iterator.next().right; // FIXME travese to the end in trie
```

Comment:
@mike-tr-adamson do you know how to force trie to traverse to the end? so it doesn't have to iterate all ids.